js/src/button.js

```diff
@@ -118,10 +88,33 @@ class Button {
 
   // Static
 
+  static _update(e) {
```

Comment:
I haven't seen this style elsewhere in the bootstrap code, but there were several code-paths with similar logic, and in the process of trying to unpick and debug it, I ended up here.